tests/test_user_pref_controller.py

```diff
@@ -1,51 +1,82 @@
-import pytest
+""" Test File for user_pref_controller file """
 import logging
-import sqlite3
 import pathlib
+# import sqlite3
 from typing import Any, Generator
+import pytest
 from src.user_pref_controller import UserPrefController
 from src.user_pref_model import UserPrefModel
 
 logger = logging.getLogger(__name__)
 logger.debug("UnitTests: UserPrefModel")
 
+
 class TestUserPrefController:
     """ Test Class for UserPrefController """
     @pytest.fixture(scope="function")
-    def access(self, tmp_path:pathlib.Path) -> Generator[UserPrefController, Any, None]:
-        """ Fixture to set up the Database environment for the tests to run in """
-        yield UserPrefController(database_location=tmp_path, database_name="test_db.db")
-
+    def access(
+            self,
+            tmp_path: pathlib.Path
+    ) -> Generator[UserPrefController, Any, None]:
+        """
+        Fixture to set up the Database environment for the tests to run in
+        """
+        yield UserPrefController(
+            database_location=tmp_path,
+            database_name="test_db.db"
+        )
 
     @pytest.fixture(scope="function")
-    def fix_create_dir(self, access:UserPrefController) -> 'Generator[UserPrefController, Any, None]':
+    def fix_create_dir(
```

Comment:
a fixture doesn't need to be prefixed with the phrase fix_. 
fix usually implies that something is being corrected.